### PR TITLE
Jd/add initial time

### PR DESCRIPTION
**Pull request by jd-lara**
Thanks for opening a PR to SIIP-PACKAGE.jl, please take note of the following when making a PR:

Check the [contributor guidelines](https://nrel-siip.github.io/SIIP-PACKAGE.jl/stable/code_base_developer_guide/developer/)

### CODE REVIEW

File: src/simulation.jl [hunk 1]
```diff
@@ -119,8 +126,10 @@ function run_simulation(simulation::ApiServer.Simulation, output_dir, channels)
     end
 
     try
-        build_status = PSI.build!(sim; console_level=Logging.AboveMaxLevel)
+        build_status = PSI.build!(sim; console_level=Logging.Info)
```
Comment (jd-lara) [inline src/simulation.jl:129]: @daniel-thom this is useful to keep while we develop more

File: python_client/swagger_client/api_client.py [hunk 2]
```diff
@@ -74,7 +74,7 @@ def __init__(self, configuration=None, header_name=None, header_value=None,
         # Set default User-Agent.
         self.user_agent = 'Swagger-Codegen/1.0.0/python'
 
-    def close(self):
+    def __del__(self):
```
Comment (daniel-thom) [inline python_client/swagger_client/api_client.py:77]: The python client code should not be here, right? If it is, we need to make sure that the scripts/openapi/build_python_client.sh generates it. That script changes `__del__` to `close` to fix a bug.

File: python_client/swagger_client/models/simulation.py [hunk 3]
```diff
@@ -31,28 +31,33 @@ class Simulation(object):
...
         self._num_steps = None
+        self._initial_time = None
         self.discriminator = None
         self.name = name
         self.models = models
         self.sequence = sequence
         if num_steps is not None:
             self.num_steps = num_steps
+        if initial_time is not None:
+            self.initial_time = initial_time
```
Comment (daniel-thom) [inline python_client/swagger_client/models/simulation.py:60]: Something has gone wrong here. It's weird that there is self._initial_time and self.initial_time.

File: src/simulation.jl [hunk 4]
```diff
@@ -78,9 +78,16 @@ function make_simulation(sim::ApiServer.Simulation, output_dir)
         )(),
     )
 
+    if !isnothing(sim.initial_time)
+        ini_time = parse(DateTime, sim.initial_time, dateformat"y-m-d HH:MM:SS")
+    else
+        ini_time = nothing
```
Comment (daniel-thom) [inline src/simulation.jl:84]: You can delete this.

File: python_client/swagger_client/models/simulation.py (hunk 3 above)
Comment (jd-lara) [reply python_client/swagger_client/models/simulation.py:60]: I was thinking the same

**Pull request merged**